lib/features/about_us/pages/about_us_page.dart

```diff
@@ -9,41 +11,39 @@ import 'package:zognest_website/shared/widgets/drawer.dart';
 import 'package:zognest_website/shared/widgets/footer.dart';
 import 'package:zognest_website/shared/widgets/image_text.dart';
 
+import '../../../config/theme/text_theme.dart';
 import '../../../resources/strings.dart';
 import '../../../shared/widgets/mouse_animation.dart';
+import '../../../shared/widgets/scroll_headline.dart';
 
-class AboutUsPage extends StatefulWidget {
-  const AboutUsPage({super.key});
-
+class AboutUsPage extends HookWidget {
+   AboutUsPage({super.key});
   static const route = '/about-us';
+  late final ScrollController controller;
```

Comment:
remove this controller there is already scroll controller used by hooks in build